pkgs/applications/version-management/radicle-cli/default.nix

```diff
@@ -2,86 +2,38 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, pkg-config
-, cmake
-, installShellFiles
-, asciidoctor
 , DarwinTools
-, openssl
-, libusb1
 , AppKit
-, git
-, openssh
-, testers
-, radicle-cli
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "radicle-cli";
-  version = "0.6.1";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "radicle-dev";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-LS6zYpMg0LanRL2M8ioGG8Ys07TPT/3hP7geEGehwxg=";
+    repo = "heartwood";
+    rev = "b27313bc5c703c8a25343176c620c8652b815abc";
+    sha256 = "sha256-WyLqTOF1TOuRD4/aTikeOR446fd00opryiUnAdPfA9Q=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "automerge-0.0.2" = "sha256-MZ1/rca8ZsEUhd3bhd502PHlBbvqAOtnWFEdp7XWmYE=";
-      "automerge-0.1.0" = "sha256-dwbmx3W13oZ1O0Uw3/D5Z0ht1BO1PmVVoWc/tLCm0/4=";
-      "cob-0.1.0" = "sha256-ewPJEx7OSr8X6e5QJ4dh2SbzZ2TDa8G4zBR5euBbABo=";
-      "libusb1-sys-0.6.2" = "sha256-577ld1xqJkHp2bqALNq5IuZivD8y+VO8vNy9Y+hfq6c=";
-      "walletconnect-0.1.0" = "sha256-fdgdhotTYBmWbR4r0OMplOwhYq1C7jkuOdhKASjH+Fs=";
-    };
-  };
-
-  # Otherwise, there are errors due to the `abigen` macro from `ethers`.
-  auditable = false;
+  cargoHash = "sha256-ZKIGaHTIvlkVIPhuGOYTG0zXBOZoNxtcNIVZJyuNa+8=";
 
-  nativeBuildInputs = [
-    pkg-config
-    cmake
-    installShellFiles
-    asciidoctor
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+  nativeBuildInputs = [ ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     DarwinTools
   ];
 
-  buildInputs = [
-    openssl
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    libusb1
+  buildInputs = [ ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     AppKit
   ];
 
-  postInstall = ''
-    for f in $(find . -name '*.adoc'); do
```

Comment:
Maybe we can revert this change since we have now again .adoc files.
Thanks to this commit

https://github.com/radicle-dev/heartwood/commit/af28a0ca07c2a0601e39b14fe3054ecb3f9a3bbc